ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/viewcreation/valueobjects/ViewConfig.java

```diff
@@ -1,34 +1,41 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects;
 
-import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
-@Configuration
-@ConfigurationProperties
+import static org.apache.commons.lang3.Validate.notNull;
+
 public class ViewConfig {
 
 	public static final String DEFAULT_VIEW_NAME = "by-page";
 	public static final String DEFAULT_VIEW_FRAGMENTATION_STRATEGY = "pagination";
 	public static final Map<String, String> DEFAULT_VIEW_FRAGMENTATION_PROPERTIES = Map.of("memberLimit", "100",
 			"bidirectionalRelations", "false");
-	private List<ViewSpecification> views = new ArrayList<>();
+	private final List<ViewSpecification> views;
 	private boolean defaultView;
 
+	public ViewConfig(List<ViewSpecification> views, boolean defaultView) {
+		this.views = views != null ? views : new ArrayList<>();
+		this.defaultView = defaultView;
+	}
+
 	public List<ViewSpecification> getViews() {
+		return getViews(null);
+	}
```

Comment:
Is this method needed? Doesn't seem to be used.

Reply:
forgot to remove that, was part of the transitioning :)